### ISSUE TITLE
Google Fonts is distributing maliciously constructed archives

### ISSUE THREAD

**Issue by ctrlcctrlv** [google/fonts#6371]
**Describe the issue**
* Ark, and other archivers based on KIO¹, report e.g. [https://github.com/notofonts/symbols/releases/download/NotoSansSymbols2-v2.006/NotoSansSymbols2-v2.006.zip](NotoSansSymbols2-v2.006.zip) to be a malicious archive:
  
  > ark.kerfuffle: Possibly malicious archive. Detected entry that could lead to a directory traversal attack: "../AUTHORS.txt"
  
* After looking into the issue² I agree.³

**To Reproduce**
self-evident

**Expected behavior**
the archive not to be malicious

**Additional context**
```
[fred@デブ狸 ~]$ plasmashell --version
plasmashell 5.27.5
[fred@デブ狸 ~]$ ark --version
ark 23.07.70
[fred@デブ狸 ~]$ crc32 Downloads/NotoSansSymbols2-v2.006.zip
38866eb1
[fred@デブ狸 ~]$ uname -a
Linux debutanuki 6.3.6-zen1-1-zen #1 ZEN SMP PREEMPT_DYNAMIC Mon, 05 Jun 2023 15:12:42 +0000 x86_64 GNU/Linux

```

**Screenshots**
![image](https://github.com/google/fonts/assets/838783/ea28cf11-b409-4f36-a6e2-7b516b831eed)

# Footnotes

¹ KDE Input/Output
² See typescript: [ark.gdb.session.tar.gz](https://github.com/google/fonts/files/11736091/ark.gdb.session.tar.gz)
³ Having `..` in a zip archive is self-evidently malicious behavior even if other unarchivers ignore it.

**Comment by davelab6** [comment]
@simoncozens looks like a trivial bug in your Noto release packager. 

@ctrlcctrlv thanks for reporting :) next time please file Noto project bugs on the Noto project issue tracker, as this isn't an issue for this repo or the gf product/service

**Comment by simoncozens** [comment]
Yeah, this is my fault; it's part of the way Noto fonts are being packaged for release. I can assure you that [Hanlon's razor](https://en.wikipedia.org/wiki/Hanlon's_razor) applies, and this is stupid rather than malicious; the problem is this line:

https://github.com/notofonts/notobuilder/blob/215db873ae49f9ac719786f4f3b2fd824da05716/.github/workflows/build.yaml#L328

I'll fix it.

**Comment by ctrlcctrlv** [comment]
thanks for closing. i also think that's a bug in Info-ZIP. Info-ZIP should be smart enough to not do it that way.

**Comment by ctrlcctrlv** [comment]
i submitted a bug report to Info-ZIP

> I think Info-ZIP should be modified never to write ".." as a ZIP archive entry.
> 
> Context is available in https://github.com/google/fonts/issues/6371.
> 
> The errant command was:
> 
> zip -x requirements.txt -r ../${{ github.ref_name }}.zip ${{ steps.check_tag.outputs.family}}/ ../*.txt ../DESCRIPTION.en_us.html ../ARTICLE.en_us.html
> 
> This causes KIO based unarchivers like Ark to declare the archives malicious.
> 

![image](https://github.com/google/fonts/assets/838783/b8641a05-fe45-48fd-a275-7d6cce6caf5a)
